sdk/src/main/java/com/kin/ecosystem/poll/presenter/PollWebViewPresenter.java

```diff
@@ -92,13 +92,7 @@ public void onResponse(OpenOrder response) {
 			@Override
 			public void onFailure(KinEcosystemException exception) {
 				showToast(SOMETHING_WENT_WRONG);
-				String errorMsg;
-				try {
-					errorMsg = ((ApiException) exception.getCause()).getResponseBody().getMessage();
-				} catch (Throwable error) {
-					errorMsg = error.getMessage();
-				}
-
+				String errorMsg = exception.getMessage();
```

Comment:
I think the best way for handling unexpected errors in log is to print the entire stacktrace to get as much data as needed using android `Log.getStackTraceString()`, it will print also the cause.
You can see an example here https://github.com/kinecosystem/atn-android/blob/0e4c0269783a82c2bfa1773159cd239ea530ef06/atnlib/src/main/java/com/kik/atn/AndroidLogger.java#L57 (I did a fix there aa Android don't print `UnknownHostException` and we did want to know about it)

Reply:
I'm getting the error msg its enough for us at this moment, this is not the case we want to see the stacktrace. later on, we will have an exception handler that will post the stacktrace to our endpoint, it's an open task :)

Reply:
:+1: